game/game.py

```diff
@@ -354,14 +353,14 @@ def finish_turn(self, skipped: bool = False) -> None:
 
     def begin_turn_0(self) -> None:
         self.turn = 0
-        self.initialize_turn()
+        self.initialize_turn(for_red=True, for_blue=True)
```

Comment:
nit: these are the defaults, so no need to specify them

Reply:
for sure! i thought it is more clearer and better readable for someone other than me and you who look into this one